### ISSUE TITLE
[Feature] Improve the usage of variables in localization strings

### ISSUE THREAD

**Issue by PineappleSnackz** [BetterDiscord/BetterDiscord#1652]
### Before Requesting

- [X] I found no existing issue matching my feature request

### Describe the feature you'd like!

English usually differs from other languages, in this case from Swedish.

For example, strings like "You don't have any {{type}}s!" become difficult to translate, as the plural form in Swedish can vary, just like irregular noun plurals in English.

Another case is strings such as "Open {{type}} Folder". The Swedish language uses compound nouns which occasionally require an interfix, which makes it hard to localize this since not all compound nouns require this interfix. 

A solution could be to limit variable usage to things such as names, time, and numbers, that won't affect the sentence structure significantly.

**Comment by rauenzi** [comment]
Yeah this is my own fault for trying to half-bake a homegrown solution for translations. I didn't have the basic foresight for these cases. This is something that will have to sit on the backlog until we refactor the way translations work to use something more standardized that helps handle these cases.

**Comment by zerebos** [comment]
The new i18n system was added with proper pluralization support! We also moved our translations over to Crowdin https://translate.betterdiscord.app/
